Update Python agent-config (#2272)
open-telemetry/opentelemetry.io: content/en/docs/instrumentation/python/automatic/agent-config.md

Here's an explanation of what each configuration does:

- `traces_exporter` specifies which traces exporter to use. In this case, traces
  are being exported to `console` (stdout) and with `otlp`. The `otlp` option
  tells `opentelemetry-instrument` to send the traces to an endpoint that accepts OTLP
  via gRPC. In order to use HTTP instead of gRPC, add `--exporter_otlp_protocol http`.
  The full list of available options for traces_exporter can be found
  [here](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/opentelemetry-instrumentation).
- `metrics_exporter` specifies which metrics exporter to use. In this case,
  metrics are being exported to `console` (stdout). It is currently required for
  your to specify a metrics exporter. If you aren't exporting metrics, specify

>   `none` as the value instead.

Author: Not sure if it's still required to set a metrics exporter (probably not). Could also remove this part.

Reviewer: At least as of last December it was needed (in my own testing). Otherwise you'd get an error at runtime. I'd love it if this isn't required anymore, though!

> - `service_name` sets the name of the service associated to the telemetries, and is

Reviewer: ```suggestion
- `service_name` sets the name of the service associated with your telemetry, and is
```

  sent to your [Observability backend](/ecosystem/vendors/).

> - `exporter_otlp_endpoint` sets the endpoint which the telemetries are exported to.

Reviewer: ```suggestion
- `exporter_otlp_endpoint` sets the endpoint where telemetry is exported to.
```
Note that "telemetries" is incorrect, as "telemetry" refers to instrumentation data already (and can be plural).
